clientv3/client.go

```diff
@@ -93,8 +93,11 @@ func NewFromURL(url string) (*Client, error) {
 // Close shuts down the client's etcd connections.
 func (c *Client) Close() error {
 	c.cancel()
-	c.Watcher.Close()
+	closeErr := c.Watcher.Close()
```

Comment:
This line is breaking `TestGRPCResolver` and `TestGRPCResolverMulti` because it changes behaviour of `integration.ClusterV3.Terminate()`...

Anyway, current `Client.Close()` ignores an error from `c.Watcher.Close()`, do you have good idea for receiving the watcher error? @heyitsanthony @xiang90